### PR TITLE
[IMP] runbot: add a small test for _local_pg methods

### PR DESCRIPTION
**Pull request by d-fence**
Small test to prevent annoying deployment

### CODE REVIEW

**Comment by d-fence** [comment]
@KangOl any idea of an assert that could check that the db is created/dropped ?

**Comment by KangOl** [comment]
You should at least connect to the database and make a query (leaving the cursor open)

To ensure the database is gone, you can search for it in `pg_database` table (with local cursor)

I would also use a random database name like `str(uuid.uuid4())`

File: runbot/tests/test_build.py [hunk 1]
```diff
@@ -58,6 +59,24 @@ def test_build_cmd_log_db(self, mock_grep, mock_mkdir):
...
 
+    def test_build_local_pg(self):
+        """ test create and drop of a local database even with an open cursor"""
+        dbname = str(uuid.uuid4())
+        self.Build._local_pg_createdb(dbname)
+        cnx = psycopg2.connect("dbname=%s" % dbname)
+        cur = cnx.cursor()
+        cur.execute("CREATE TABLE test (id serial PRIMARY KEY, foo varchar);")
+        cur.execute("INSERT INTO test (foo) VALUES ('bar')")
+        cnx.commit()
```
Comment (KangOl) [inline runbot/tests/test_build.py]: Isn't it better to keep the cursor open?

**Comment by Xavier-Do** [comment]
Creating a database in a test is a bad idea, this should be at least -standard. Anyway, this was open for a while without being merged, can we close this? :)

**Pull request closed without merging**